### ISSUE TITLE
is it possible to clear app cache? **Question**

### ISSUE THREAD

**Issue by muccagelato** [kzahel/web-server-chrome#64]
when i ran my website first it had a favicon image, but now i want to change it so i deleted the old icon image, and i put in a new one, but the browser still shows me the same old image, i alredy deleted the browser cache, can you please tell me how can i delete the app one?
anyway your app is great.

**Comment by kzahel** [comment]
Hmm I thought I disabled this.
Are you serving from a google drive folder? Sometimes there are caching problems when serving from google drive.

**Comment by muccagelato** [comment]
No, I chose a folder from my desktop...

Il 20/ago/2016 06:32 PM, "Kyle Graehl" <notifications@github.com> ha
scritto:

**Comment by muccagelato** [comment]
Also changing the name of the icon image the browser still read the old one.

**Comment by kzahel** [comment]
Hmm. Well I disabled caching a long time ago, so the app should not be caching. However there might be some other issue. Are you using windows?

You might be able to restart the web server by pressing Ctrl-R or Cmd-R. If that doesn't work you can go to chrome://flags and enable "Debugging for packed apps" which will let you right click in the web server window and click on "reload app"

**Comment by muccagelato** [comment]
I 've just tried, and i restarted the computer too.
Yeah i'm on windows.

**Comment by kzahel** [comment]
Another thing you might try is opening Chrome devtools and going to options. You should see an option for "Disable cache while devtools is open" in there somewhere...

**Comment by muccagelato** [comment]
I tried from my page and reload but it doesn't work, or you mean from the
app?

**Comment by kzahel** [comment]
No i mean from your page. In Devtools settings
<img width="831" alt="screen shot 2016-08-20 at 10 01 38 am" src="https://cloud.githubusercontent.com/assets/158352/17832661/1ead515e-66bd-11e6-9759-9a5d70719cfb.png">

**Comment by muccagelato** [comment]
Yeah i alfredy founded it

**Comment by muccagelato** [comment]
Okay, it doesn't work

**Comment by kzahel** [comment]
Does the same thing happen with using another web server? Not sure what other web servers are easy to install on windows... if you have python installed you could do python -m SimpleHTTPServer

**Comment by muccagelato** [comment]
Well, thank you for your time, tell me if you find an answer.
Very good app, again.

**Comment by muccagelato** [comment]
No, i've only chrome, but tommorow  il will try as you said and i'll let you know.
Sorry but now i've to go.
Bye

**Comment by muccagelato** [comment]
hi, 
I've just load my website using hostingmywebsite.com and ftp , and the icon is displayed correctly,
but your app is still showing the old image.
is there anything that I can do?

**Comment by muccagelato** [comment]
hey, I don't know why but now the icon display correctly,
i just went on with the website.

**Comment by yusufmsalh** [comment]
@kzahel  any update regarding disabling the cache ,I am facing a similar situation ,as I loading a video from local path ,when I change the video(replace a mew  with old video name ),I still see the old video.

**Comment by kzahel** [comment]
Caching was disabled long ago -- check your devtools options and make sure caching is disabled.
It's possible there's a bug on the HTML5 FS API somehow using the old entry. If you can reproduce the problem consistently, make a screen recording or write down the steps and I can look into it.

**Comment by yusufmsalh** [comment]
Step 1 : Create an HTMl file ,with video tag, Set Source to
Http://127.x.y.z/1.mp4,
Where 1.mp4 is a  local Video seen by your server.
Step 2: while video 1 is playing, create another video with same name at
your directory and delete the old one.
Step 3: Refresh Your Html page.
Expected Output: New Video is being played.
Current Output: old video  is being played,
Also I tried clearing Cache ,disable it,going private mode, nothing,
Only solved by opening a new tab and writing page url in it.
Kindly Waiting for your Response.

**Comment by yusufmsalh** [comment]
Hi there Mr.Fekry ,Hope you are Fine.

**Comment by ethanaobrien** [comment]
The browser is caching the favicon, not the app. Clear cache in the browser
